src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
+    private User user;
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist): When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value.
```suggestion
    private Long id;
    private List<Long> ticketIds;
    private User user;
```